resources/views/navbar/profile.blade.php

```diff
@@ -1,6 +1,13 @@
+@php($showDarkMode = config('ui.dark-mode.enabled') === true && ($profileMenuDarkMode ?? false))
+
 <x-ark-dropdown
     wrapper-class="whitespace-nowrap md:relative"
-    :dropdown-classes="'w-full md:w-auto mt-4 '.($profileMenuClass ?? null)"
+    :dropdown-classes="Arr::toCssClasses([
+        'mt-4',
+        'w-full md:w-auto' => ! $showDarkMode,
+        'w-64'             => $showDarkMode,
```

Comment:
do we need a width or a min-width? 🤔

Reply:
:thinking: min-width might be more appropriate for here yeh. Will update :+1: